Makefile

```diff
@@ -1,6 +1,9 @@
 OS=$(shell uname)
 VERSION=v1.12.0
-CGO_LDFLAGS="-Lgen/third_party/rplidar_sdk-release-${VERSION}/sdk/output/${OS}/Release/"
+GIT_REVISION = $(shell git rev-parse HEAD | tr -d '\n')
+TAG_VERSION?=$(shell etc/tag_version.sh)
+CGO_LDFLAGS="-L 'gen/third_party/rplidar_sdk-release-${VERSION}/sdk/output/${OS}/Release/'"
```

Comment:
[q] why did this line change? i.e. removal of the space and addition of quotes. Was it previously wrong?

Reply:
It was not, I just figure this was easier to read.